src/Types.ts

```diff
@@ -539,6 +542,14 @@ export interface RequireUint256LowerBoundRequest extends RequireCheckRequest {
 export interface RequireUint256LowerBoundResponse extends RequireCheckResponse {
   oracleValue: string
 }
+export interface RequireUint256UpperBoundRequest extends RequireCheckRequest {
```

Comment:
these request and response classes were intended to be used in API, but I don't think API is using them. I think we should not add these for now. I think these will be necessary if we build a TypeScript API client, but that's not super high priority yet